pyls/workspace.py

```diff
@@ -236,16 +229,18 @@ def jedi_script(self, position=None):
         sys_path = self.sys_path(environment_path) + extra_paths
         environment = self.get_enviroment(environment_path) if environment_path else None
 
+        project_path = self._workspace.root_path if self._workspace else os.path.dirname(self.path)
         kwargs = {
-            'source': self.source,
+            'code': self.source,
             'path': self.path,
-            'sys_path': sys_path,
             'environment': environment,
+            'project': jedi.Project(path=project_path, sys_path=sys_path),
+                                        sys_path=sys_path),
```

Comment:
There's a clear syntax error here.

Reply:
On it. Leave it alone for a few minutes.

Reply:
Ok